src/einsteinpy/utils/schwarzschild_utils.py

```diff
@@ -21,6 +20,8 @@ def schwarzschild_radius(mass):
         Schwarzschild radius for a given mass
 
     """
+    if not isinstance(mass, u.quantity.Quantity):
```

Comment:
Great what you did there! 
Could you add two more parameters to the function `schwarzschild_radius(mass, c=constant.c, G= constant.G)` and do the same with the other two parameters as you did with `mass`. Change `num` and `deno` variables according to the change.